tsl/src/compression/compression.c

```diff
@@ -678,7 +679,8 @@ row_compressor_append_row(RowCompressor *row_compressor, TupleTableSlot *row)
 		if (compressor == NULL)
 			continue;
 
-		// TODO since we call getallatts at the beginning, slot_getattr is useless
+		// Performance Improvement: Since we call getallatts at the beginning, slot_getattr is
+		// useless
 		//     overhead here, and we should just access the array directly
```

Comment:
```suggestion
		/* Performance Improvement: Since we call getallatts at the beginning, slot_getattr is
		   useless overhead here, and we should just access the array directly */
```